packages/core/src/browser/tree/tree-model.ts

```diff
@@ -44,6 +44,10 @@ export interface ITreeModel extends ITree, ITreeSelectionService, ITreeExpansion
      */
     openNode(node?: ITreeNode | undefined): void;
     /**
+     * Event for when the user double clicks a node.
+     */
+    readonly onOpenNode: Event<ITreeNode | undefined>;
```

Comment:
Strange that a node can be `undefined`. I don't think clients are expecting it.

Reply:
Ok, will change it. The idea was to keep it in sync with the contract of `openNode` but that's not important.